src/status_im/ui/components/action_button/styles.cljs

```diff
@@ -15,40 +9,40 @@
    :ios            {:height 63}
    :android        {:height 56}})
 
-(defnstyle action-button-icon-container [cyrcle-color]
+(defnstyle action-button-icon-container [circle-color]
   {:border-radius    50
    :width            40
    :height           40
    :align-items      :center
    :justify-content  :center
-   :ios              {:background-color (or cyrcle-color color-light-blue-transparent)}})
+   :ios              {:background-color (or circle-color styles/color-light-blue-transparent)}})
 
 (def action-button-label-container
   {:padding-left 16})
 
 (defstyle action-button-label
-  {:ios     {:color          color-light-blue
+  {:ios     {:color          styles/color-light-blue
```

Comment:
why not colors ns?

Reply:
I didn't migrate all colors yet. It would make sense to have a task just for that.

Reply:
#3140